pkg/sdn/plugin/subnets.go

```diff
@@ -95,11 +95,6 @@ func (master *OsdnMaster) deleteNode(nodeName string) error {
 	if err != nil {
 		return fmt.Errorf("Error fetching subnet for node %q for deletion: %v", nodeName, err)
 	}
-	_, ipnet, err := net.ParseCIDR(sub.Subnet)
-	if err != nil {
-		return fmt.Errorf("Error parsing subnet %q for node %q for deletion: %v", sub.Subnet, nodeName, err)
-	}
-	master.subnetAllocator.ReleaseNetwork(ipnet)
```

Comment:
I'm not describing the scenario clearly :)

t=0, node exists, subnet exists
t=1, node is deleted
t=2, node is created (tries to allocate/create new subnet? gets AlreadyExists error, keeps existing subnet?)
t=3, node deletion event is delivered
t=4, deleteNode() looks up and deletes the subnet
t=5, watchSubnets() sees the subnet deletion event, calls DeleteHostSubnetRules

it looks like we can get in a situation where we have a node without an allocated subnet. is that the case? how does the systme recover from that?